appserver/batch/hazelcast-jbatch-store/src/main/java/fish/payara/jbatch/persistence/hazelcast/HazelcastPersistenceService.java

```diff
@@ -56,8 +56,8 @@ and Distribution License("CDDL") (collectively, the "License").  You
     private IMap jobInstanceMap;
     private IMap checkpointMAP;
     private HazelcastInstance theInstance;
-    private IdGenerator jobInstanceIdGenerator;
-    private IdGenerator checkpointIdGenerator;
+    private FlakeIdGenerator jobInstanceIdGenerator;
+    private FlakeIdGenerator checkpointIdGenerator;
```

Comment:
Btw, could you kill all these comments?
` //To change body of generated methods, choose Tools | Templates.`